app/config/webpack/common.js

```diff
@@ -2,16 +2,16 @@ const path = require('path');
 const webpack = require('webpack');
 const StyleLintPlugin = require('stylelint-webpack-plugin');
 
-const rootDir = path.resolve(__dirname, '../../../');
+const projectDir = path.resolve(__dirname, '../../../');
```

Comment:
this rename is not needed as it's in the JS, but I did it for consistency with back-end naming